filament/backend/src/vulkan/VulkanContext.h

```diff
@@ -43,6 +43,7 @@ struct VulkanCommandBuffer;
 struct VulkanAttachment {
     VulkanTexture* texture = nullptr;
     uint8_t level = 0;
+    uint8_t baseViewIndex = 0;
     uint16_t layer = 0;
 
```

Comment:
we do need a layerCount here unfortunately.  I might think about how to refactor this later, but for now, we need it to know the correct VkImageSubresourceRange